src/asmcnc/apps/start_up_sequence/screens/screen_release_notes.py

```diff
@@ -144,12 +150,43 @@ def __init__(self, **kwargs):
         self.sm = kwargs["screen_manager"]
         self.version = kwargs["version"]
         self.l = kwargs["localization"]
-        # Filename consists of just the version digits followed by .txt, so can be found by filtering out non integers from version name
-        # Two dots before filename mean parent directory, as file is at the top of the filetree, not in src
-        self.release_notes_filename = "../" + self.version.replace(".", "") + ".txt"
-        self.scroll_release_notes.release_notes.source = self.release_notes_filename
+        self.model_manager = kwargs["router_machine"].model_manager
+        machine_type = self.model_manager.get_machine_type()
+        self.scroll_release_notes.release_notes.source = self.get_release_notes_source(machine_type)
         self.update_strings()
 
+        # Remove the knowledgebase link & QR code if the machine is not a SmartBench
+        if machine_type is not MachineType.SMARTBENCH:
+            self.release_notes_and_links.remove_widget(self.release_links)
+
+    def get_release_notes_source(self, type=None, MachineType=MachineType):
+        """
+        This method is used to generate the source path for the release notes file.
+
+        Parameters:
+        type (Enum): The type of the machine. If the type is 'unknown', it defaults to looking for a generic release notes file.
+        MachineType (Enum): The enum containing the machine types.
+
+        Returns:
+        str: The path to the release notes file. The file name is generated by concatenating the version number (with dots removed),
+             the machine type (if not 'unknown'), and the '.txt' extension. The path is then constructed by joining the easycut path and the file name.
+
+        Example:
+        If the version is 'v2.9.0' and the machine type is 'unknown', the resulting file name would be 'v290.txt'.
+        If the version is 'v2.9.0' and the machine type is 'SmartBench', the resulting file name would be 'v290_SmartBench.txt'.
+        """
+        version_string = self.version.replace(".", "")  # v2.9.0 -> v290
+        file_extension = ".txt"
+
+        if type == MachineType.UNKNOWN or type is None:
+            # Machine type not found, look for generic release notes
+            target_file_name = version_string + file_extension  # v290.txt
+        else:
```

Comment:
Am I missing the distinction for DRYWALLTEC ?

Reply:
Machine type (if valid) goes into the filename. See line 186

Reply:
See, I was right. I missed it. :)